app/src/main/java/com/android/streetworkapp/ui/tutorial/TutorialSignIn.kt

```diff
@@ -0,0 +1,176 @@
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.DirectionsRun
+import androidx.compose.material.icons.filled.ConnectWithoutContact
+import androidx.compose.material.icons.filled.LocationOn
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.android.sample.R
+import com.android.streetworkapp.ui.authentication.IconAndTextRow
+import com.android.streetworkapp.ui.theme.ColorPalette
+
+@Composable
+fun TutorialSignIn() {
+  // Set number of page
+  val pagerState = rememberPagerState(pageCount = { 3 })
+  Box(
+      contentAlignment = Alignment.Center,
+      modifier = Modifier.fillMaxWidth().testTag("introScreenBoxContainer")) {
+        HorizontalPager(state = pagerState) { page ->
+          // Display the corresponding page
+          when (page) {
+            0 -> IntroPage1()
+            1 -> IntroPage2()
+            2 -> IntroPage3()
+          }
+        }
+      }
+  // Dots indicator
+  Row(
+      horizontalArrangement = Arrangement.Center,
+      verticalAlignment = Alignment.CenterVertically,
+      modifier = Modifier.padding(16.dp).testTag("introDotRow")) {
+        repeat(3) { index ->
+          val isSelected = pagerState.currentPage == index
+          Box(
+              modifier =
+                  Modifier.size(if (isSelected) 12.dp else 12.dp)
+                      .padding(4.dp)
+                      .clip(CircleShape)
+                      .background(
+                          if (isSelected) ColorPalette.INTERACTION_COLOR_DARK
+                          else ColorPalette.BORDER_COLOR))
+        }
+      }
+}
+
+@Composable
+fun IntroPage1() {
+  Column(
+      modifier = Modifier.fillMaxWidth().testTag("introColumn1"),
+      verticalArrangement = Arrangement.Center,
+      horizontalAlignment = Alignment.CenterHorizontally) {
+        Box(modifier = Modifier.fillMaxWidth().testTag("introBox1")) {
+
+          // Background image
+          Image(
+              painter = painterResource(id = R.drawable.intro_1),
```

Comment:
Docs: I think it would be great to add the source of the different images used. (Or a small note saying that they are AI generated, if they are)

Reply:
Do i add it inside the code as a comment ?

Reply:
Added the source